tests/unit/components/Popover/Popover.spec.js

```diff
@@ -42,7 +42,11 @@ describe('Popover.vue', () => {
 			expect(wrapper.find('.content').isVisible()).toBe(false)
 		})
 		it('is open when prop is set', async () => {
+			const div = document.createElement('div')
+			div.id = 'root'
+			document.body.appendChild(div)
 			const wrapper = mount(Popover, {
+				attachTo: '#root',
```

Comment:
shouldn't this be done via the `container` prop?

Reply:
`attachTo` is an option from @vue\test-utils -> mount function, see https://v1.test-utils.vuejs.org/api/options.html#attachto It specifies to which HTMLElement the component will be mounted to.
`container` is an option of the `Popover` component itself, specifying where the popover menu will be attached to. The popover should open no matter the `container` prop (but it also doesn't open if I specify it).
I am a bit lost to be honest.

Reply:
🤔